src/devtools/client/inspector/rules/components/Declarations.tsx

```diff
@@ -14,11 +14,9 @@ export const Declarations: FC<DeclarationsProps> = ({ declarations, query }) =>
 
   return (
     <ul className="ruleview-propertylist">
-      {declarations
-        .filter(declaration => !declaration.isInvisible)
```

Comment:
Because we render `Rule`s slightly different between the three categories (pseudo, style, inherited), this wasn't filtering sufficiently all of the rules, so that's why the filtering was moved to be a responsibility of the `RulesApp`.